internal/containersuite/containerSuite.go

```diff
@@ -216,10 +216,6 @@ func (s *ContainerSuite) SetupSuite() {
 	}
 
 	ctx := context.Background()
-	err = validateImageExists(ctx, s.ContainerImage)
```

Comment:
not needed as we always pull, on camunda/camunda a locally built image was used thus no pull was necessary